cmd/orgs.go

```diff
@@ -215,6 +233,148 @@ func orgsRemove(ctx *cli.Context) error {
 	return nil
 }
 
+func orgsMembersListCmd(ctx *cli.Context) error {
+
+	// Use "member" team name to bypass specific teams
+	teamName := "member"
+
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
+
+	var getMembers sync.WaitGroup
+	getMembers.Add(2)
+
+	var org *envelope.Org
+	var orgs []envelope.Org
+	var team envelope.Team
+	var teams []envelope.Team
+	var memberships []envelope.Membership
+	var oErr, tErr, mErr, sErr error
+
+	// Retrieve the org name supplied via the --org flag.
+	// This flag is optional. If none was supplied, then
+	// orgFlagArgument will be set to "". In this case,
+	// prompt the user to select an org.
+	orgFlagArgument := ctx.String("org")
+
+	if orgFlagArgument == "" {
+		// Retrieve list of available orgs
+		orgs, oErr = client.Orgs.List(c)
+		if oErr != nil {
+			return oErr
+		}
+
+		idx, _, oErr := SelectOrgPrompt(orgs)
+		if oErr != nil {
+			return oErr
+		}
+
+		if idx == promptui.SelectedAdd {
+			// COME BACK TO THIS
+			// Error condition, unsure how to handle
+		}
+
+		org = &orgs[idx]
+	}
+
+	go func() {
+		// If org flag was used, identify the org supplied.
+		if orgFlagArgument != "" {
+			org, oErr = client.Orgs.GetByName(c, orgFlagArgument)
+			if org == nil {
+				oErr = errs.NewExitError("Org not found.")
+				return
+			}
+		}
+
+		// Retrieve the team by name supplied
+		teams, tErr = client.Teams.GetByName(c, org.ID, teamName)
+		if len(teams) != 1 {
+			tErr = errs.NewExitError("Team not found.")
+			getMembers.Done()
+			return
+		}
+		team = teams[0]
+
+		// Hide machine teams from the teams list; as we use them to represent
```

Comment:
You don't need this bit of code here as you are always looking up the `members` team

Reply:
We can also dry up this code a bit too, as this and the team members command are doing a lot of the same work.

Reply:
You're right, that was clumsy, will remove. Also agree that I'm duplicating some code from `teams members` so after this `orgs` cmd is working, can re-factor to avoid that.